Frontend/ui-customer/src/Components/DriverInfoCard/View.purs

```diff
@@ -265,6 +266,7 @@ supportButton push state =
   , height WRAP_CONTENT
   , orientation VERTICAL
   , gravity CENTER
+  , visibility if ((state.props.currentStage == RideAccepted) || (state.props.currentStage == RideStarted) || (state.props.currentStage == ChatWithDriver)) && (not state.props.showChatNotification) then VISIBLE else GONE
```

Comment:
@PraveenGongada can we create a function which would return this value as the same condition is being used at many places.

Reply:
Done